guides/appendix/configuration.md

```diff
@@ -13,15 +13,31 @@ would be `liveReload` in the configuration file.
   "liveReload": false
 }
 ```
+
 Every development environment will be different but a realistic example setting `--port` and `--proxy` is shown below:
 
 ```json {data-filename=.ember-cli}
 {
   // disableAnalytics added by ember new
-  "disableAnalytics": false, 
-  "port" : 8080,
+  "disableAnalytics": false,
+  "port": 8080,
   "proxy": "http://localhost:3000"
 }
 ```
 
 For a complete list of command line options run `ember help`.
+
+### `package.json` Configuration
+
+Some configuration is exposed through your `package.json` file.
+
+If you have a nested repo structure (e.g., a monorepo using yarn workspaces) and want to allow `ember s` from the root of the repo, you can configure your `package.json` to look like:
+
+```json {data-filename=package.json}
+{
+  "name": "top-level-workspaces-repo",
+  "ember-addon": {
+    "projectRoot": "./packages/path-to-ember-application"
```

Comment:
I haven't used a nested repo yet. To confirm, would the placeholder path `./packages/path-to-ember-application` be clear enough to most developers, or should we provide a more concrete example?

Reply:
Hmm, good point. I think it's _probably_ clear enough, but we could add some more text explaining that `./packages/path-to-ember-application` is the relative path to where Ember project lives (eg, the dir where `ember-cli-build.js` exists). Thoughts?

Reply:
An additional text that explains your knowledge would help, yep!